src/components/Homepage/events/MobileElements.js

```diff
@@ -19,7 +19,7 @@ export const MobileMeetups = ({ events }) => (
         {events.map(conf => (
           <li key={`${conf.id}`}>
             <Subtitle noPaddingBottom>
-              <ExternalAnchor href={conf.linkToEvent}>
+              <ExternalAnchor href={conf.linkToEvent} title={conf.linkToEvent}>
```

Comment:
Do we really want the title to be the same as href (rather than conference name, or something)?

Reply:
good point, will change

Reply:
good point, will change

Reply:
updated